src/Microsoft.AspNetCore.Mvc.Core/CompatibilityVersion.cs

```diff
@@ -66,7 +66,15 @@ public enum CompatibilityVersion
         /// <remarks>
         /// ASP.NET Core MVC 2.2 introduces compatibility switches for the following:
         /// <list type="bullet">
+        ///     <item><description><c>ApiBehaviouOptions.SuppressMapClientErrors</c></description></item>
```

Comment:
While you're here, could you update the 2.1 list too:

``
MvcOptions.AllowCombiningAuthorizeFilters
                    MvcOptions.AllowBindingHeaderValuesToNonStringModelTypes
                    MvcOptions.AllowValidatingTopLevelNodes
                    MvcOptions.InputFormatterExceptionPolicy
                    MvcOptions.SuppressBindingUndefinedValueToEnumType
```